TestHelpers.Utils.Tests/CreateA/Date.cs

```diff
@@ -0,0 +1,94 @@
+﻿using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestHelpers.Utils;
+using Xunit;
+
+namespace CreateATests
+{
+    public class Date
+    {
+        [Fact]
+        public void GivenTwoDaysBeforeADate_ReturnsADateTwoDaysBeforeSuppliedDate()
```

Comment:
What do you think about trying to be consistent in the use of **Given**[Some pre-condition]**_Then**[Some expected result]?